.gitignore

```diff
@@ -1,5 +1,5 @@
 # generated by users and intentionally not pushed to the repo
```

Comment:
This is weird. pip-tools based workflows are specifically built around having "lock files" checked in for reproducibility..

Reply:
to make this legit @jakob-keller we should have one lockfile per python version and delete this cmt and txt

Reply:
Not just per python but per each permutation of python+OS+arch. This is what extensive locking looks like: https://github.com/cherrypy/cheroot/tree/main/requirements. However, to implement such an integrated environment, a lot of effort is required, so I'd postpone.

Reply:
I was aware of that and agree that it should be addressed at some point. As it stands, we could throw out pip-tools from CI/CD and just run `pip install .[awscli,boto3] -r requirements-dev.in` or similar. But I suggest to leave it as it is for now and have a separate effort to improve that aspect, maybe even switch dependency management tool again (poetry, hatch, ...).